content/3_Silver/Greedy_Sorting.problems.json

```diff
@@ -106,7 +106,7 @@
       "name": "Stick Division",
       "url": "https://cses.fi/problemset/task/1161",
       "source": "CSES",
-      "difficulty": "Hard",
+      "difficulty": "Normal",
```

Comment:
yeah no

Reply:
i mean this is difficulty, not how good it is

Reply:
Good = difficulty 🙂